### ISSUE TITLE
Request pipeline icons no longer working

### ISSUE THREAD

**Issue by kbrandwijk** [graphcool/console#1222]
**Do you want to request a *feature* or report a *bug*?**  
Bug

**What is the current behavior?**
If I create a new RP hook, the icon in the function list no longer highlights the hook point.
![image](https://user-images.githubusercontent.com/852069/28248148-e1824924-6a3f-11e7-8cee-7840c42a22dc.png)

**What is the expected behavior?**
Icon should highlight the hook point.

**Which browser and OS are affected by this issue? Did this work in previous versions of the Console?**
Worked before.

**Comment by timsuchanek** [comment]
Yes, I disabled them, because we have a problem with Relay Classic, where it doesn't handle interfaces correctly and tries to extend fragments of interfaces into each other that are not allowed to. As soon as we migrated to Relay Modern, we can enable them again.

**Comment by kbrandwijk** [comment]
So now I can also no longer see what Type and action (C,U,D) the RP is for. The icon is no big problem, because you can see which RP hook it is when opening it, but the action is not visible anywhere.
